deploy/docker/Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM registry.access.redhat.com/ubi7-minimal
+FROM registry.access.redhat.com/ubi7-minimal AS base-amd64
+FROM registry.access.redhat.com/ubi8-minimal AS base-arm64
+
+FROM base-${TARGETARCH:-amd64}
```

Comment:
For switch base image.

default value here to support `docker buildx build` && `docker build` both.

`docker build` could not pass arg `TARGETARCH` in `FROM`

Reply:
`s390x` or `ppc64le` support could just add.
```
FROM registry.access.redhat.com/ubi8-minimal AS base-s390x
FROM registry.access.redhat.com/ubi8-minimal AS base-ppc64le
```